### PR TITLE
Fix table node styles(Align column types, adjust handle position, and round column corners in tables)

### PR DESCRIPTION
**Pull request by FunamaYukina**
## Summary

I have modified the styles for TableNodes.

## Changes

The following three styles have been changed.

### First
https://github.com/liam-hq/liam/pull/206/commits/905da8b198904ad88eb66cbab79ed8e9a291d353 : add border-radius to the columnWrapper last-child

Before --> After

ss 2467

### Second

https://github.com/liam-hq/liam/pull/206/commits/126362cdf2fc17fb422c96c9d8aef6db3e7818fa : align column type names to the center

Before --> After

ss 2464

### Third

https://github.com/liam-hq/liam/pull/206/commits/6b5b799c3715d6cfdb954d6221a920bea69b4aa1 : adjust handle position

Before --> After

ss 2465

### CODE REVIEW

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.module.css [hunk 1]
```diff
@@ -23,6 +23,10 @@
   border-bottom: 1px solid var(--node-border);
 }
 
+.columnWrapper:last-child {
+  border-radius: var(--border-radius-md);
```
Comment (MH4GF) [inline frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.module.css:27]: As the upward radius is not needed, the following may be acceptable:

```suggestion
  border-bottom-left-radius: var(--border-radius-md);
  border-bottom-right-radius: var(--border-radius-md);
```

https://developer.mozilla.org/en-US/docs/Web/CSS/border-bottom-left-radius

**Review by MH4GF — COMMENTED** [review]
COOOL!!!!!!!!!
I've wrote a small comment!

File: frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.module.css (hunk 1 above)
Comment (FunamaYukina) [reply frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.module.css]: Oh, exactly...🤦‍♀️.Thank you!
Fixed: https://github.com/liam-hq/liam/pull/206/commits/81a7c5e2e4522f004dca426ff57e07545bd23c06

**Review by MH4GF — APPROVED** [review]
LGTM 🚀

**Pull request merged**